src/EA.Weee.Web.Tests.Unit/Areas/Scheme/ViewModels/ManageEvidenceNotesDisplayOptionsTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace EA.Weee.Web.Tests.Unit.Areas.Scheme.ViewModels
+{
+    using Core.Helpers;
+    using FluentAssertions;
+    using Web.Areas.Scheme.ViewModels.ManageEvidenceNotes;
+    using Xunit;
+
+    public class ManageEvidenceNotesDisplayOptionsTests
+    {
+        [Fact]
+        public void ManageEvidenceOverviewDisplayOption_ShouldHaveCorrectDisplayAttributes()
+        {
+            //assert
+            ManageEvidenceNotesDisplayOptions.Summary.ToDisplayString().Should().Be("evidence-summary");
```

Comment:
is that summary tab will be ever used? not part of your PR but just saw it's never used

Reply:
Yeah it will be at some point